platform_plugin_ontask/api/v1/views.py

```diff
@@ -77,23 +86,41 @@ def post(self, request, course_id: str) -> Response:
             api_auth_token = get_api_auth_token(course_block)
 
             ontask_client = OnTaskClient(settings.ONTASK_INTERNAL_API, api_auth_token)
-            response = ontask_client.create_workflow(course_id)
+            create_workflow_response = ontask_client.create_workflow(course_id)
 
-            if not response.ok:
-                log.error(f"POST {response.url} | status-code={response.status_code} | response={response.text}")
+            if not create_workflow_response.ok:
+                log.error(ontask_log_from_response(create_workflow_response))
                 return Response(
                     data={
-                        "error": "An error occurred while creating the workflow. Ensure the "
-                        "workflow for this course does not already exist, and that the OnTask "
-                        "API Auth token is correct."
+                        "error": "An error occurred while creating the workflow. "
+                        "Ensure the workflow for this course does not already "
+                        "exist, and that the OnTask API Auth token is correct."
                     },
                     status=status.HTTP_400_BAD_REQUEST,
                 )
 
-            created_workflow = response.json()
-            course_block.other_course_settings["ONTASK_WORKFLOW_ID"] = created_workflow["id"]
+            workflow_id = create_workflow_response.json()["id"]
+            course_block.other_course_settings["ONTASK_WORKFLOW_ID"] = workflow_id
             update_item(CourseKey.from_string(course_id), course_block, request.user.id)
 
+            enrollments = get_user_enrollments(course_id)
+            data_frame = {"user_id": {}}
+            for idx, enrollment in enumerate(enrollments):
+                data_frame["user_id"][idx] = enrollment.user.id
+
+            update_table_response = ontask_client.update_table(workflow_id, data_frame)
+
+            if not update_table_response.ok:
+                log.error(ontask_log_from_response(update_table_response))
+                return Response(
+                    data={
+                        "error": "An error occurred while updating the table. "
+                        "Ensure the workflow for this course exists, and that "
+                        "the OnTask API Auth token is correct."
+                    },
```

Comment:
Why not:
```suggestion
                     "An error occurred while updating the table. "
                     "Ensure the workflow for this course exists, and that "
                     "the OnTask API Auth token is correct."
```

Reply:
Done: https://github.com/eduNEXT/platform-plugin-ontask/commit/898eeca07dd4d409d7299805a1b9a218884121c7